STEPS.md

```
@@ -101,32 +101,41 @@ Sets complex configuration.
 
 ## DrupalContext
 
-[Source](src/Drupal/DrupalExtension/Context/DrupalContext.php)
+[Source](src/Drupal/DrupalExtension/Context/DrupalContext.php), [Example](tests/behat/features/drupal.feature)
 
 >  Provides pre-built step definitions for interacting with Drupal.
```

markdownlint-cli2 (0.22.1)

[warning] Multiple spaces after blockquote symbol

(MD027, no-multiple-space-blockquote)